SymphonyApi/ArmoniK.DevelopmentKit.SymphonyApi/api/SessionPollingService.cs

```diff
@@ -192,30 +194,30 @@ public IEnumerable<string> SubmitTasks(IEnumerable<byte[]> payloads)
         taskRequests.Add(taskRequest);
       }
 
-      TaskHandler.CreateTasksAsync(taskRequests,
-                                   TaskOptions).Wait();
+      var createTaskReply = TaskHandler.CreateTasksAsync(taskRequests,
+                                                         TaskOptions).Result;
 
-      var taskCreated = taskRequests.Select(t => t.Id);
+      switch (createTaskReply.ResponseCase)
+      {
+        case CreateTaskReply.ResponseOneofCase.None:
+          throw new Exception("Issue with Server !");
+        case CreateTaskReply.ResponseOneofCase.CreationStatusList:
+          Logger.LogInformation("Tasks created : {ids}",
```

Comment:
à encapsuler dans un test sur le niveau de log

Reply:
out of scope ?

Reply:
Just add a `if(Logger.IsEnabled(Information)` to avoid executing the `string.Join` function everytime.